### ISSUE TITLE
Distinction between "Delete" and "Remove"

### ISSUE THREAD

**Issue by shirayuki** [Tampermonkey/tampermonkey-i18n#237]
To maintain translation consistency, could you consider distinguishing between "delete" and "remove" or standardizing their usage?

```
"message": "Do you really want to permanently delete all userscripts?"
"message": "Remove local script \"$name$\" ($uuid$)",
"message": "Remove remote script \"$name$\" ($uuid$)",
"message": "The script was successfully deleted."
"message": "This script was deleted"
"message": "This script was deleted by the hoster"
```

**Comment by maboroshin** [comment]
I have a few opinions as well.

----

I think "Word wrap" is more common term in text editors. e.g. Notepad on Windows, Notepad++. 

* Original : ```Auto Line break```
* Suggestion : ```Word wrap``` or ```Wrap lines```

----

"Discard changes" is one of the BIOS menu items. It is easy to understand for those who understand BIOS. For discarding changes that are not factory reset, it would be better to use "discard". VioletMonkey also uses the word "Discad" for this dialog. Following string:

* Original : ```Really reset all changes?```
* Suggestion : ```Really discard all changes?```

* Original : ```Editor reset```
* Suggestion : ```Discard changes```

**Comment by derjanb** [comment]
> could you consider distinguishing between "delete" and "remove" or standardizing their usage?

@shirayuki OK, I'll change all to "Delete".

>  it would be better to use "discard"

@maboroshin  Yes, "discard" sounds good to me.

> Auto Line break

I may add a minus: "Automatic line-break"

Thanks to both of you.

**Comment by derjanb** [comment]
Fixed by 7da759adf00774894f55a9b4b03e2d4f0d1c7c1a
